app/manifest_name_to_discover_name.py

```diff
@@ -5003,4 +5003,6 @@
             'files/source/HB-ICN-Ch2-1362238629-Raw_Data.csv': 'files/source/HB-ICN-Ch2-1362238629-Raw Data.csv',
             'files/source/HB-ICN-Ch4-1362238626-Raw_Data.csv': 'files/source/HB-ICN-Ch4-1362238626-Raw Data.csv', 'files/source/manifest_.xlsx': 'files/source/manifest.xlsx',
             'files/source/HB-ICN-Ch3-1362238627-Raw_Data.csv': 'files/source/HB-ICN-Ch3-1362238627-Raw Data.csv',
-            'files/source/HB-ICN-CH1-1362238630-Raw_Data.csv': 'files/source/HB-ICN-CH1-1362238630-Raw Data.csv'}
+            'files/source/HB-ICN-CH1-1362238630-Raw_Data.csv': 'files/source/HB-ICN-CH1-1362238630-Raw Data.csv',
+            'files/derivative/sam-P21 BAT/ses-Pgp9.5/P21-3_Pgp9.5_-_Movie.mp4': 'files/derivative/sam-P21 BAT/ses-Pgp9.5/P21-3 Pgp9.5 - Movie.mp4'
```

Comment:
Actually it looks like the mapping is in the file, but the path was either rnamed or entered in incorrectly initially (I believe the latter bc the second part of the mappings are correct).

If you search this file for: 'files/derivative/sam-P21 BAT/ses-Pgp9.5/P21-3 Pgp9.5 - Movie.mp4' you will see that there is already a mapping for it. However, it's path to map from is incorrect. It contains 'ses-Pgp9' instead of 'ses-Pgp9.5'. You should remove this line that you added and instead search the file for all occurrences of 'ses-Pgp9' and replace it with 'ses-Pgp9.5'

Reply:
done